src/main/scala/cromwell/backend/model/ExecutionResult.scala

```diff
@@ -32,5 +32,4 @@ case class FailureResult(exception: Throwable) extends ExecutionResult
   * @param overallHash Contains a hash from TaskDescriptor.
   * @param dockerHash Contains docker image hash.
   */
-// TODO: remove dockerHash.
-case class ExecutionHash(overallHash: String, dockerHash: Option[String])
\ No newline at end of file
+case class ExecutionHash(overallHash: String, dockerHash: Option[String] = None)
```

Comment:
OK. But dockerHash should be part of overallHash (that's why overallHash has its name right?)